docs/source/use-cases.rst

```diff
@@ -0,0 +1,375 @@
+Use Cases
+=========
+
+Dask is a versatile tool that supports a variety of workloads.  This page
+contains brief and illustrative examples for how people use Dask in practice.
+This page emphasizes breadth and hopefully inspires readers to find new ways
+that Dask can serve them beyond their original intent.
+
+Overview
+--------
+
+Dask algorithms cases can be roughly divided in the following two categories:
+
+1.  Large NumPy/Pandas/Lists with :doc:`dask.array<array>`,
+    :doc:`dask.dataframe<dataframe>`, :doc:`dask.bag<bag>` to analyze large
+    datasets with familiar techniques.  This is similar to Databases, Spark_,
+    or big array libraries.
+2.  Ad-hoc task scheduling.  You submit a graph of functions that depend on
+    each other for custom workloads.  This is similar to Luigi_, Airflow_,
+    Celery_, or Makefiles_.
+
+Most people today approach Dask assuming it is a framework like Spark, designed
+for the first use case around large collections of uniformly shaped  data.
+However, many of the more productive and novel use cases fall into the second
+category, using Dask to parallelize custom workflows.
```

Comment:
Part of me wants this "," to be a ":", but that might not be correct grammar either.